### ISSUE TITLE
Can't read skript-vers.json

### ISSUE THREAD

**Issue by sparrowhe** [liz3/SkIDE#3]
**Describe the bug**
![image](https://user-images.githubusercontent.com/26871028/103542089-5f601680-4ed7-11eb-84b8-1a73c01bbb2c.png)
like this👆

**To Reproduce**
1. run SkIDE.exe
2. see the bug

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS: win32
 - Version 2020.1.3

**Additional context**
I can visit https://skripttools.net/api.php?t=skript&action=getlist, but SkIDE may can't

**Comment by RauchF** [comment]
Experiencing the same issue, on Windows 10 64bit, with version 2020.1.3 after running the tool linked in issue #1

**Comment by liz3** [comment]
@RauchF @sparrowhe yes there was a issue with a resource URL returning 403...its fixed now, rerun the updater tool to 2021.0.2 to get the fixed version

**Comment by liz3** [comment]
@sparrowhe the resources server didnt like the request did not contain a http user agent header leading to the 403.
https://github.com/liz3/SkIDE/commit/24709cf43c8a52a2425ead891a46baa7c68c179c
